pool_lib/__init__.py

```diff
@@ -1181,49 +1207,54 @@ def sort(self, key, reverse: bool = False):
 
     @classmethod
     def init_create(
-            cls: Type['Pool'],
+            cls: Type["Pool"],
             buildroot: AnyPath,
-            path: Optional[AnyPath] = None) -> 'Pool':
+            path: AnyPath | None = None
+    ) -> "Pool":
 
         if path is None:
             cwd = os.getcwd()
-            path = os.path.normpath(os.getenv('POOL_DIR', cwd))
+            path = os.path.normpath(os.getenv("POOL_DIR", cwd))
             pool_path = join(realpath(path), ".pool")
             if not isdir(pool_path):
                 path = cwd
 
-        spath = join(realpath(str_path(path)), '.pool')
-        path_pkgcache = join(spath, 'pkgcache')
-        path_stocks = join(spath, 'stocks')
-        path_tmp = join(spath, 'tmp')
-        path_build = join(spath, 'build')
-        path_build_root = join(spath, 'build/root')
-        path_build_logs = join(spath, 'build/logs')
-        path_build_info = join(spath, 'build/buildinfo')
-        path_srcpkgcache = join(spath, 'srcpkgcache')
+        spath = join(realpath(str_path(path)), ".pool")
+        path_pkgcache = join(spath, "pkgcache")
+        path_stocks = join(spath, "stocks")
+        #path_tmp = join(spath, "tmp")
```

Comment:
Line can be removed.

Reply:
recommendation applied in https://github.com/turnkeylinux/pool/pull/18/commits/3bb007db690b5b8b8ca6cb272d6e577a88ad5364